src/component.js

```diff
@@ -180,8 +181,11 @@ class Component {
   }
 
   async filterComponents({parents, depth, limit, installedOnly}, children = []) {
+    const id = this.id === '' ? '/' : this.id;
+    const {name, version} = utils.unpackId(id);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "name" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'name' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)